src/config/defaultConfig.js

```diff
@@ -1,32 +1,32 @@
 import nextConfig from 'next/config';
-
 import { AccountId } from 'utils/deploymentInfo';
 
-const accountId = nextConfig()?.publicRuntimeConfig?.accountId ?? AccountId.HMS;
-
-const supportEmailsByAccountId = {
-  [AccountId.HMS]: 'ccbhelp@hms.harvard.edu',
-};
-
-const botTokensByAccountId = {
-  [AccountId.HMS]: 'eG94Yi0zODE4MjQwMTYwODY5LTM4Mjk3Njc0OTQ4MDQtbDJtTHF4TTNpZFRRQXZYQUNJTXdFbDBx', // pragma: allowlist secret
+const accountId = nextConfig()?.publicRuntimeConfig?.accountId ?? AccountId.BIOMAGE;
+
+const accountInfo = {
+  [AccountId.HMS]: {
+    supportEmail: 'ccbhelp@hms.harvard.edu',
+    botToken: 'eG94Yi0zODE4MjQwMTYwODY5LTM4Mjk3Njc0OTQ4MDQtbDJtTHF4TTNpZFRRQXZYQUNJTXdFbDBx', // pragma: allowlist secret
+    feedbackWebhookEndpoint: 'aHR0cHM6Ly9ob29rcy5zbGFjay5jb20vc2VydmljZXMvVDAzUTI3MjRRUksvQjAzUThCR0JLSEIvUk41elZtU1RudXUyMWxURUh1a3I3ajdJ', // pragma: allowlist secret
+    referralWebhookEndpoint: 'aHR0cHM6Ly9ob29rcy5zbGFjay5jb20vc2VydmljZXMvVDAzUTI3MjRRUksvQjAzVDdDSjFVODIvWFJ3cGtaa2huT2ZCbEVObkNMamtad0pn', // pragma: allowlist secret
+  },
+  default: {
+    supportEmail: 'hello@biomage.net',
+    botToken: 'eG94Yi0xMTc1NDc1NTM5NzE0LTMyMjE3MTYwNDg4OTgtUWFXNmlwTEVSY1RJRXQycVU2RnJ3RGp1', // pragma: allowlist secret
+    feedbackWebhookEndpoint: 'aHR0cHMlM0ElMkYlMkZob29rcy5zbGFjay5jb20lMkZzZXJ2aWNlcyUyRlQwMTU1RFpGVk0wJTJGQjAyS0ZESEc0SkslMkZFY3ZZZVhQNVBoM0lQMjE0ZlNTRWhGT3c=', // pragma: allowlist secret
+    referralWebhookEndpoint: 'aHR0cHM6Ly9ob29rcy5zbGFjay5jb20vc2VydmljZXMvVDAxNTVEWkZWTTAvQjAyQVk0ODQxQ0cvQ0x3Mms4dTBtMkUzcDBVNUhhbjBqeTBv', // pragma: allowlist secret
+  },
 };
 
-const feedbackWebhookEndpointsByAccountId = {
-  [AccountId.HMS]: 'aHR0cHM6Ly9ob29rcy5zbGFjay5jb20vc2VydmljZXMvVDAzUTI3MjRRUksvQjAzUThCR0JLSEIvUk41elZtU1RudXUyMWxURUh1a3I3ajdJ', // pragma: allowlist secret
-};
-
-const referralWebhookEndpointsByAccountId = {
-  [AccountId.HMS]: 'aHR0cHM6Ly9ob29rcy5zbGFjay5jb20vc2VydmljZXMvVDAzUTI3MjRRUksvQjAzVDdDSjFVODIvWFJ3cGtaa2huT2ZCbEVObkNMamtad0pn', // pragma: allowlist secret
-};
+const getAccountInfo = (account) => accountInfo[account] || accountInfo.default;
```

Comment:
Having Biomage or HMS as the default for account configuration should be avoided where possible. Is this is necessary for local development only where `nextConfig()` won't work? If so, can we just add an environmental variable `CELLENICS_AWS_ACCOUNT_ID` or similar that is used as the default instead and then index `accountId` by organization?

Reply:
The `AWS_ACCOUNT_ID`  is picked up originally from `next.config.js` and stored in `publicRuntimeConfig.accountId`.

Following your idea, I added a `DEV_ACCOUNT` that should be added when doing `npm run` like this:  `DEV_ACCOUNT=HMS npm run`

Reply:
With [this](https://github.com/hms-dbmi-cellenics/ui/pull/848#discussion_r1270674147) you shouldn't see biomage anymore locally. 

About the `default` you mentioned should be avoided where possible, the problem here is that we want to display `supportEmail` and all of these for more than `AccountId.BIOMAGE`, I do think that this could be moved over to `getDomainSpecificContent` instead though, let me know if you would prefer that. However, keep in mind that in `getDomainSpecificContent` the `default` is set to `BIOMAGE_PRIVATE` (so the behavior would be the same).

Reply:
I like the changes that you have proposed in https://github.com/biomage-org/ui/pull/250. While it still uses non-specified AWS account IDs (the `default` case) for Biomage private deployments, it is much better in that it contains it within a single file (`getDomainSpecificContent.jsx`), which can be readily changed in the future. Your proposed changes also remove the account id checks and organization name drops across multiple files which is greatly appreciated.

I would prefer a more descriptive environment variable just to make it clear the the account is for AWS (e.g. `CELLENICS_DEV_AWS_ACCOUNT_ID`). It is pretty long ... would it make sense to move the domain-specific content in the api to a similar centralized file like in the ui with `getDomainSpecificContent`? There are fewer items currently (only in `getAdminSub.js` and `src/default-config.js` that I can see).